cSpell.json

```diff
@@ -557,6 +557,55 @@
                 "Speciality"
             ]
         },
+        {
+            "filename": "**/specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json",
+            "words": [
+                "detectwithurl",
```

Comment:
[nit] use vscode to sort this, Ctrl+Shift+P -> Sort Lines Ascending

Reply:
fixed, and removed repeats,